README.md

```diff
@@ -1,14 +1,10 @@
-# Knative Sample Controller
+# Knative Contour KIngress controller.
 
-[![GoDoc](https://godoc.org/knative.dev/sample-controller?status.svg)](https://godoc.org/knative.dev/sample-controller)
-[![Go Report Card](https://goreportcard.com/badge/knative/sample-controller)](https://goreportcard.com/report/knative/sample-controller)
+[![GoDoc](https://godoc.org/knative.dev/net-contour?status.svg)](https://godoc.org/knative.dev/net-contour)
+[![Go Report Card](https://goreportcard.com/badge/knative/net-contour)](https://goreportcard.com/report/knative/net-contour)
 
-Knative `sample-controller` defines a few simple resources that are validated by
-webhook and managed by a controller to demonstrate the canonical style in which
-Knative writes controllers.
-
-To learn more about Knative, please visit our
```

Comment:
@mattmoor : Link back to knative original repo seems relevant. Why was this removed?